sqlserver/assets/configuration/spec.yaml

```diff
@@ -181,6 +181,15 @@ files:
       value:
         type: boolean
         example: false
+    - name: include_index_usage_metrics
+      description: |
+        Configure collection of user table index usage statistics from the sys.dm_db_index_usage_stats DMV.
+        
+        It is recommended to enable `database_autodiscovery` or set `database`, since the sys.dm_db_index_usage_stats
+        view is scoped to the current database.
```

Comment:
```suggestion
        Configure the collection of user table index usage statistics from the `sys.dm_db_index_usage_stats` DMV.
        
        Because the `sys.dm_db_index_usage_stats` view is scoped to the current database, enable 
        `database_autodiscovery` or set `database`.
```